src/components/Posts/Cards/SkeletonPostPreviewCard/SkeletonPostPreviewCard.spec.tsx

```diff
@@ -1,18 +1,10 @@
 import React from 'react';
-import { MemoryRouter } from 'react-router-dom';
-import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
 import { SkeletonPostPreviewCard } from './SkeletonPostPreviewCard';
 
-beforeEach(() =>
-  render(
-    <MemoryRouter>
-      <SkeletonPostPreviewCard />
-    </MemoryRouter>,
-  ),
-);
-
 describe('SkeletonPostPreviewCard', () => {
   it('renders the component', () => {
-    expect(screen.getByTestId('skeleton')).toBeInTheDocument();
+    const tree = renderer.create(<SkeletonPostPreviewCard />).toJSON();
```

Comment:
there were no need to pull in a test-renderer. You could return a `container` from `render` function that you deleted and match it with a spanshot